internal/blueprint/customizations.go

```diff
@@ -90,6 +91,12 @@ type FilesystemCustomization struct {
 	MinSize    uint64 `json:"minsize,omitempty" toml:"size,omitempty"`
 }
 
+type OpenSCAPCustomization struct {
+	DataDir    string `json:"data_dir,omitempty" toml:"data_dir,omitempty"`
```

Comment:
Why is this configurable? Could we hardcode a value?

Reply:
Ah yeah we could hardcode the value here. Do you have any suggestions? Maybe /var/log/openscap_data?

Reply:
Maybe use our own namespace? /var/log/osbuild/openscap? @gicmo thoughts?

Reply:
Yeah, that sounds good!